### ISSUE TITLE
parsing a  malformed tuple with a single entry to xds_from_table 

### ISSUE THREAD

**Issue by SpheMakh** [ska-sa/xarray-ms#38]
* xarray-ms version: 0.1.9
* Python version: 2.7
* Operating System: ubuntu 18.04

### Description

When a parse a malformed tuple to xds_from_table does not raise an exception but it ignores the specified entry. In the example below, the *columns* is parsed a malformed tuple `("DATA")`

```
ds = xds_from_ms("msdir/ESO137-MeerKAT_OT-FULL-01.ms", columns=("DATA"))
```

No exception is raised but the DATA column is not loaded. 

````
 Dimensions without coordinates: row
 Data variables:
     *empty*
````

When I use a proper tuple `("DATA",)` it works
```
ds = xds_from_ms("msdir/ESO137-MeerKAT_OT-FULL-01.ms", columns=("DATA",))
```

**Comment by bennahugo** [comment]
It should probably just accept strings and tuples

**Comment by sjperkins** [comment]
Column handling logic is failing in some fashion here:

https://github.com/ska-sa/xarray-ms/blob/d4c52e43f5a7e41e17d8ce821a6d12b60e0691b3/xarrayms/xarray_ms.py#L783-L795

Referencing ``group_cols`` instead of ``index_cols`` on line 788.

and  probably here:

https://github.com/ska-sa/xarray-ms/blob/d4c52e43f5a7e41e17d8ce821a6d12b60e0691b3/xarrayms/xarray_ms.py#L922-L937

The

```python
elif not isinstance(something_cols, list):
    something_cols = [something_cols]
```

probably needs to be changed to:

```python
if not isinstance(something_cols, list)
    something_cols = [something_cols]
```

**Comment by sjperkins** [comment]
Fixed by #40. Workaround till next release is to use lists of strings when supplying column names.
